recipe/build.sh

```diff
@@ -6,14 +6,7 @@ else
     SONAME="so"
 fi
 
-cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
--D TIFF_LIBRARY=$PREFIX/lib/libtiff.${SONAME} \
--D TIFF_INCLUDE_DIR=$PREFIX/include \
--D PNG_LIBRARY_RELEASE=$PREFIX/lib/libpng.${SONAME} \
--D PNG_PNG_INCLUDE_DIR=$PREFIX/include \
--D ZLIB_LIBRARY=$PREFIX/lib/libz.${SONAME} \
--D ZLIB_INCLUDE_DIR=$PREFIX/include \
-.
+cmake -D CMAKE_INSTALL_PREFIX=$PREFIX -D TIFF_LIBRARY=$PREFIX/lib/libtiff.${SONAME} -D TIFF_INCLUDE_DIR=$PREFIX/include -D PNG_LIBRARY_RELEASE=$PREFIX/lib/libpng.${SONAME} -D PNG_PNG_INCLUDE_DIR=$PREFIX/include -D ZLIB_LIBRARY=$PREFIX/lib/libz.${SONAME} -D ZLIB_INCLUDE_DIR=$PREFIX/include .
```

Comment:
Sorry, I wasn't very clear. I should have said I like the way you have done line wraps here, but it would be nice if the options line up. Like so...

    cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
          -D TIFF_LIBRARY=$PREFIX/lib/libtiff.${SONAME} \